backend/.env.example

```diff
@@ -2,8 +2,22 @@ INDABA_PG_USERNAME=indabauser
 INDABA_PG_TESTUSER=indaba
 INDABA_PG_PASSWORD=abadni
 INDABA_PG_DB=indabatestuser
+INDABA_PG_HOSTNAME=localhost
 AUTH_SALT=nMsDo)_1fh
 JWT_SECRET=0a6b944d-d2fb-46fc-a85e-0295c986cd9f
+SURVEY_SERVICE_URL=http://localhost:9005/api/v1.0/
+AUTH_SERVICE_URL=http://localhost:4000/api/v0
+DOMAIN=http://localhost:3005
+GMAIL_PASS=indabapassword
 MESSAGE_SERVICE_URL=http://localhost:4002
 SYS_MESSAGE_USER=indaba@example.com
-SYS_MESSSAGE_PASSWORD=password
+SYS_MESSAGE_PASSWORD=password
+S3_BUCKET=indaba-project-file-uploads
+AWS_ACCESS_KEY_ID=AKIAI5LWCTK3VMXKMERA
+AWS_SECRET_ACCESS_KEY=/ans0TwDIPi+fr/iWc/hEAf9XyTmVP7T0aKRfxLL
```

Comment:
Okay, crazy, I got an email from Amazon with this exact link saying my Access Keys were public online and therefore compromised. I removing ...